recipe/test_cmake.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bas
-cd ${RECIPE_DIR}/test_cmake
+cd test_cmake
```

Comment:
Was this necessary? Seems like a different issue? I guess if it doesn't hurt...

Reply:
I was confused about this too. I think there are intricacies between how the recipe dir is accessed for split recipes.

I had to declare `test_cmake` to be copied in from the recipe dir at test time.